config/spryk/spryks/Spryker/Yves/Plugin/AddSubFormPluginDataProviderMethod.yml

```diff
@@ -0,0 +1,41 @@
+spryk: method
+description: "Adds a new *module* class method."
+mode: both
+
+arguments:
+    module:
+        inherit: true
+
+    organization:
+        inherit: true
+        default: SprykerEco
+
+    paymentMethod:
+        inherit: true
+
+    method:
+        inherit: true
+        value: "createSubFormDataProvider"
+
+    input:
+        value: ""
+
+    output:
+        value: \Spryker\Yves\StepEngine\Dependency\Form\SubFormInterface
+
+    annotations:
+        value:
+            - "{@inheritdoc}"
+            - "@api"
+
+    subDirectory:
+        value: "Yves/{{ module }}/Plugin"
+
+    template:
+        value: Common/Method.php.twig
+
+    target:
+        value: "{{ organization }}\\{{ subDirectory | convertToClassNameFragment }}\\{{ paymentMethod }}SubFormPlugin"
```

Comment:
the same: generic spyk or payment related.

Reply:
done